app/jobs/organization_event_job.rb

```diff
@@ -5,21 +5,31 @@ class OrganizationEventJob < ApplicationJob
   queue_as :github_event
 
   # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/MethodLength
   def perform(payload_body)
     return true unless payload_body.dig("action") == "member_removed"
 
     github_user_id = payload_body.dig("membership", "user", "id")
     github_organization_id = payload_body.dig("organization", "id")
-    @organization = Organization.find_by(github_id: github_organization_id)
+    organizations = Organization.where(github_id: github_organization_id)
```

Comment:
We definitely want the collection here, so, we shouldn't tack the `.all?` on, otherwise, we'll be iterating over a boolean on line 18.

I think `empty?` does work here though. That query returns an ActiveRecord::Relation. If you run `.empty?` on an AR relation, it behaves as expected (if there are no records in it, it returns true, else false).

But, it seems like `.empty?` might be a non-standard thing to use here. I ended up using it because I had a vague recollection that one of the checks was less effecient than the others. Based on [this post](https://mensfeld.pl/2015/06/empty-vs-blank-vs-any-why-you-should-not-use-any-to-test-if-theres-anything-in-the-array/) it seemed `.empty?` was the way to go. That said, it's unlikely we'll be in the situation where this type of optimization will truly matter, so if `.any?` would be the more common thing to use here, I'm happy to update the check on line 16.

Reply:
Ah, if it works then it's good! Either way. I realized I had a formatting error above: `.all?` was supposed to be `.all`? (i.e., the question mark wasn't meant to be part of the method, but was definitely formatted that way 😭)